src/server/ApiMessage.h

```diff
@@ -60,7 +60,7 @@ typedef struct _CONSOLE_API_MSG
     CONSOLE_API_STATE State{};
 
     IDeviceComm* _pDeviceComm{ nullptr };
-    IApiRoutines* _pApiRoutines{ nullptr };
+    ApiRoutines* _pApiRoutines{ nullptr };

```

Comment:
Taking out IApiRoutines will screw up my FHL for making VT Passthrough. I could rework it tho.

Reply:
Reverted.